sdcm/tester.py

```diff
@@ -1079,6 +1079,12 @@ def run_stress_thread_bench(self, stress_cmd, duration=None, stress_num=1, keysp
                                 round_robin=False, stats_aggregate_cmds=True, keyspace_name=None, use_single_loader=False):  # pylint: disable=too-many-arguments,unused-argument
 
         timeout = self.get_duration(duration)
+
+        if self.params.get("authenticator") and self.params.get("authenticator") == "PasswordAuthenticator":
+            if authenticator_user := self.params.get("authenticator_user"):
+                authenticator_password = self.params.get("authenticator_password")
```

Comment:
can the authenticator password be empty?

Reply:
better 
```python
if (authenticator_user := self.params.get("authenticator_user")) and \
    (authenticator_password := self.params.get("authenticator_password")):
```

Reply:
It's not allowed to assign two variables in the if simultaneously